### ISSUE TITLE
lottie: masking drawing issue

### ISSUE THREAD

**Issue by afsalz** [thorvg/thorvg#2426]
Facing an issue in thorvg-viewer when trying to render the below animation. 

[Animation - 1718852817326.json](https://github.com/user-attachments/files/15908817/Animation.-.1718852817326.json)

![Screenshot 2024-06-20 at 12 29 57 PM](https://github.com/thorvg/thorvg/assets/128362946/b7df047d-cfdb-423e-b517-823078619e46)

**Comment by tinyjin** [comment]
Thanks @afsalz we're going to check

**Comment by hermet** [comment]
```
[Switching to Thread 0x7fffeee9b640 (LWP 17637)]
tvg::Paint::Impl::composite (this=0x7fffb0011930, source=0x7fffb00247e0, target=0x7fffb0011970, method=<optimized out>) at ../src/renderer/tvgPaint.h:113
113                     P(compData->target)->unref();
(gdb) where
#0  tvg::Paint::Impl::composite (this=0x7fffb0011930, source=0x7fffb00247e0, target=0x7fffb0011970, method=<optimized out>)
    at ../src/renderer/tvgPaint.h:113
#1  0x00007ffff7ebc313 in tvg::Paint::composite (this=<optimized out>, target=..., method=<optimized out>) at ../src/renderer/tvgPaint.cpp:442
#2  0x00007ffff7f1aed2 in _updateMaskings (exps=0x0, frameNo=52.5676003, layer=0x7fffe0000e80) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1173
#3  _updateLayer (exps=0x0, frameNo=52.5676003, layer=0x7fffe0000e80, root=0x7fffe0027630) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1224
#4  _updateLayer (root=0x7fffe0027630, layer=0x7fffe0000e80, frameNo=52.5676003, exps=0x0) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1200
#5  0x00007ffff7f1ad2c in _updateMatte (exps=0x0, frameNo=52.5676003, layer=0x7fffe0008370, root=0x7fffe0027630)
    at ../src/loaders/lottie/tvgLottieBuilder.cpp:1186
#6  _updateLayer (exps=0x0, frameNo=52.5676003, layer=0x7fffe0008370, root=0x7fffe0027630) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1222
#7  _updateLayer (root=0x7fffe0027630, layer=0x7fffe0008370, frameNo=52.5676003, exps=0x0) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1200
#8  0x00007ffff7f1af41 in _updatePrecomp (exps=0x0, frameNo=<optimized out>, precomp=0x7fffe0027630)
    at ../src/loaders/lottie/tvgLottieBuilder.cpp:1001
#9  _updateLayer (exps=0x0, frameNo=<optimized out>, layer=0x7fffe0027630, root=0x7fffe0027300) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1228
#10 _updateLayer (root=0x7fffe0027300, layer=0x7fffe0027630, frameNo=<optimized out>, exps=0x0) at ../src/loaders/lottie/tvgLottieBuilder.cpp:1200
#11 0x00007ffff7f1b6b2 in LottieBuilder::update (this=0x555555919b30, comp=<optimized out>, frameNo=<optimized out>)
    at ../src/loaders/lottie/tvgLottieBuilder.cpp:1380
#12 0x00007ffff7f206b6 in LottieLoader::run (this=0x5555559410c0, tid=<optimized out>) at ../src/loaders/lottie/tvgLottieLoader.cpp:37
#13 0x00007ffff7ec008c in tvg::Task::operator() (tid=14, this=0x555555941138) at ../src/renderer/tvgTaskScheduler.h:66
#14 tvg::TaskSchedulerImpl::run (this=0x555555590460, i=13) at ../src/renderer/tvgTaskScheduler.cpp:153
#15 0x00007ffff759b253 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#16 0x00007ffff732aac3 in start_thread (arg=<optimized out>) at ./nptl/pthread_create.c:442
#17 0x00007ffff73bc850 in clone3 () at ../sysdeps/unix/sysv/linux/x86_64/clone3.S:81
```

**Comment by tinyjin** [comment]
@afsalz The crash has fixed in main branch. Will be included in 0.14

**Comment by hermet** [comment]
@afsalz the fix will be in v0.14.1
